clang/lib/Driver/Driver.cpp

```diff
@@ -6677,3 +6678,134 @@ llvm::Error driver::expandResponseFiles(SmallVectorImpl<const char *> &Args,
 
   return llvm::Error::success();
 }
+
+namespace {
+
+const char *GetStableCStr(std::set<std::string> &SavedStrings, StringRef S) {
```

Comment:
We prefer `static` for functions

https://llvm.org/docs/CodingStandards.html#anonymous-namespaces

Reply:
fixed